js/interface.js

```diff
@@ -58,6 +60,10 @@ function getAppsList() {
     }]);
     // Add to HTML
     apps.forEach(addApps);
+
+    if (data && data.appId) {
+      $('[data-app-id="' + data.appId + '"][data-browse-folder], [data-app-id="' + data.appId + '"] .list-holder').trigger('click');
```

Comment:
can we call the function directly instead of triggering a click?

Reply:
@seromenho Thanks, I have refactored the code to just call the functions.